packages/react/src/internal/components/Announce.tsx

```diff
@@ -44,7 +44,7 @@ export function Announce({children, politeness = 'polite', ...rest}: AnnouncePro
     // When the text of the container changes, announce the new text
     const observer = new MutationObserver(mutationList => {
       for (const mutation of mutationList) {
-        if (mutation.type === 'characterData') {
+        if (mutation.type === 'characterData' || mutation.type === 'childList') {
```

Comment:
I'd love your opinion here @joshblack! 

When I added `<Status>` initially to `FormControl`, I realized that the `live-region` element was not being populated. This seems to be because when `slots.validation` is true, the validation region is rendered within the DOM which the mutation observer does not seem to be catching. This appears to be due to it not being a change that would fall under `characterData`, as elements are either appended within `<Status>` or removed. Using `childList` catches this and populates `live-region` when the contents within are rendered or removed.

Not sure if there is a better way to go about this, or if this was intended behavior 🤔

Reply:
Oh whoa 🤯 I totally thought that `characterData` would change if the child list would change 🤔 

Maybe something we can do on mutation is to include things like childList so that it gets triggered correctly and we just check manually to see if the message has changed? Let me know if you'd want to pair on this btw!

Reply:
That's a good idea, that way if contents within change but there's no movement in terms of the text content, we would only call `announceFromElement` if the text content has changed between mutations. I'll dig on this a bit, but I think this would be a good opportunity to pair!